build/xtask/src/config.rs

```diff
@@ -9,14 +9,39 @@ use std::path::{Path, PathBuf};
 
 use anyhow::{anyhow, bail, Context, Result};
 use indexmap::IndexMap;
-use serde::Deserialize;
+use serde::{Deserialize, Serialize};
 
 use crate::auxflash::{build_auxflash, AuxFlash, AuxFlashData};
 use lpc55_areas::{
     BootSpeed, CFPAPage, CMPAPage, DebugSettings, DefaultIsp, RKTHRevoke,
     ROTKeyStatus, SecureBootCfg,
 };
 
+/// An `PatchedConfig` allows a minimal form of inheritance between TOML
+/// files.  Specifically, it allows you to **add features** to specific tasks;
+/// nothing else.
+///
+/// Here's an example:
+/// ```toml
+/// name = "sidecar-a-lab"
+/// inherit = "rev-a.toml"
```

Comment:
Does this need a `[patches]` section?

Reply:
Yup, I forgot to update that when I tweaked the syntax.